apps/jobs/serializers.py

```diff
@@ -111,10 +118,25 @@ def to_representation(self, instance):
                             }
                         }
                     )
+                else:
```

Comment:
Did you not take latest pull from the repo? Because this code snippet isn't a part of latest updated code. Always take a pull before you start making changes. 

This latest code snippet doesn't contain an else block. 
Check this : 

```python
if data:
            # Extract the URLs from social handles
            try:
                if instance.social_handles:
                    found_url_patterns = findall(
                        "((https?:\/\/)?[\w\.\/?=]+)", data.pop("social_handles", "")
                    )
                    if found_url_patterns:
                        instance.social_handles = [url[0] for url in found_url_patterns]

                data.update(
                    {
                        "Contact": {
                            "Address": data.pop("address", None),
                            "Phone": data.pop("phone", None),
                            "Website": data.pop("website", None),
                            "Email": data.pop("email", None),
                            "Social Handles": instance.social_handles,
                        }
                    }
                )

            except Exception as err:
 ```